### DISCUSSION TITLE
How to set permission in folder/dashboard as code?

### DISCUSSION THREAD

**Discussion by sdwerwed** [grafana/grafana#46378] [General]
Hello, in docs https://grafana.com/docs/grafana/latest/permissions/dashboard-folder-permissions/ does not say how I could set permissions with IaaC. I use Kubernetes and can not change the permission in the dashboard as there is no documentation on how to do that, the change I do in the dashboards are temporarily as a pod restart will revert that 

Any idea how could I manage permissions?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
